src/Components/index.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import * as glob from 'glob';
+import * as path from 'path';
+
+const options = { cwd: __dirname, ignore: './*.test.js' };
+const files = glob.sync('./*.js', options);
+
+describe('Components', () => {
+  let i;
+
+  for(i = 0; i < files.length; i++) {
```

Comment:
`file.forEach`?

Reply:
Yea I always had `forEach` async issues....I dunno why...but I don't with `map`....is that weird?

Reply:
Maybe it's related to this?
https://coderwall.com/p/kvzbpa/don-t-use-array-foreach-use-for-instead

Being slower? I'm sure if this is true, u can't tell most of the time. Most of the time when I experience this it isn't a small array either so shrug.